BookingService/BookindService.API/appsettings.json

```diff
@@ -0,0 +1,17 @@
+{
+  "Logging": {
+    "LogLevel": {
+      "Default": "Information",
+      "Microsoft.AspNetCore": "Warning"
+    }
+  },
+  "AllowedHosts": "*",
+  "ConnectionStrings": {
+    "DefaultConnection": "Data Source=DESKTOP-VM12JTU;Initial Catalog=BookingService;Integrated Security=True;TrustServerCertificate=True;Encrypt=False"
+  },
+  "Jwt": {
+    "Key": "my_super_super_super_super_secret_key",
+    "Issuer": "MyAuthServer",
+    "Audience": "MyAuthClient"

```

Comment:
Issuer and audience will be address of your site. First is the one who created token, second on is the one who this token is for. So specify for debug: http://localhost:port